tests/system/test_system.py

```diff
@@ -218,10 +218,40 @@ def test_vector_search_collection_with_filter(client, database):
     }
 
 
+@pytest.mark.skipif(FIRESTORE_EMULATOR, reason="Require index and seed data")
+@pytest.mark.parametrize("database", [None, FIRESTORE_OTHER_DB], indirect=True)
+def test_vector_search_collection_with_distance_parameters(client, database):
+    # Documents and Indexes are a manual step from util/bootstrap_vector_index.py
+    collection_id = "vector_search"
+    collection = client.collection(collection_id)
+
+    vector_query = collection.find_nearest(
+        vector_field="embedding",
+        query_vector=Vector([1.0, 2.0, 3.0]),
+        distance_measure=DistanceMeasure.EUCLIDEAN,
+        limit=3,
+        distance_result_field="vector_distance",
+        distance_threshold=1.0,
+    )
+    returned = vector_query.get()
+    assert isinstance(returned, list)
+    assert len(returned) == 2
+    assert returned[0].to_dict() == {
+        "embedding": Vector([1.0, 2.0, 3.0]),
+        "color": "red",
+        "vector_distance": 0.0,
+    }
+    assert returned[1].to_dict() == {
+        "embedding": Vector([2.0, 2.0, 3.0]),
+        "color": "red",
+        "vector_distance": 1.0,
+    }
+
```

Comment:
It might be good to have some different values of distance_threshold/distance_measure in these tests. (You could use `@pytest.mark.parametrize` to test various arguments/expected results in the same test function)

Reply:
This was a bit tricky because not all the distance measures have the same results, for some of them I was able to make it work, for a couple others I just duplicated the bulk of the test.